src/app/app.server.ts

```diff
@@ -0,0 +1,33 @@
+// Import express module and Express data type.
+import express, { Express, Router } from "express"
+import { router as cakesRouter } from "./cakes/cakes.router"
+import { router as salesRouter } from "./sales/sales.router"
+
+// Init express server. (Inferred datatype)
+const app: Express = express()
+
+//Environment Variables Init
+require('dotenv').config()
+
+// Init express router.
+const router: Router = Router()
+
+// Declare diffeerent routes.
+router.use('/cakes', cakesRouter)
+router.use('/sales', salesRouter)
+
+// Load new routers to redirection module inside app.
+app.use(router)
+app.get('/', function (req, res) {
```

Comment:
You can add this route to the main router, just to have all the routes in the same object. But like this, it works too.

Reply:
Oh, ok. So the best approach would be to have a 'app.router.ts' file? 
Also, could you help me figure which other lines I might need to move to that separate file or is it just the '/' route?

Reply:
Of course. If you create a file `app.router.ts` you can add all the routes and router configurations, just to have a single file responsibility files

`app.server.ts` -> Creates an HTTP server with express and connects the main router to it.
`app.router.ts` -> Creates a main router for the application and connects the entity's routers to it.

I think I will be uploading into a branch my example on how to solve the first challenge. Still deciding 😛